schema_builder.py

```
@@ -0,0 +1,372 @@
+                    "question": f"Which {city} restaurants have kids menus?",
+                    "answer": f"All of our recommended family-friendly restaurants in {city} offer dedicated kids menus with favorites like chicken fingers, pasta, and grilled cheese."
+                },
+                {
+                    "question": f"Do these restaurants have high chairs available?",
+                    "answer": f"Yes, every restaurant on our family-friendly list provides high chairs and booster seats for young diners."
+                }
+            ],
+            'quick-lunch': [
+                {
+                    "question": f"How fast is service at these {city} lunch spots?",
+                    "answer": f"Most restaurants on our quick lunch list can serve your meal within 15-20 minutes, perfect for busy lunch breaks."
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)